### PR TITLE
Update subnav to remove breadcrumb and just keep versions and search

### LINKED ISSUE #29336: docs breadcrumb tweaks
First of all, I think we should move `icons/booticon-chevron-right.svg` to Sass.

Secondly, I think we should properly implement breadcrumbs.

Thirdly, right now we don't follow our own [accessibility guidelines](https://twbs-bootstrap.netlify.com/docs/4.3/components/breadcrumb/#accessibility)

About the second part, I'll see what I can do with Hugo.

### PR DESCRIPTION
**Pull request by mdo**
Screen Shot 2019-09-03 at 9 09 31 AM

Closes #29336.

Preview: https://deploy-preview-29368--twbs-bootstrap.netlify.com/

### CODE REVIEW

File: site/layouts/partials/docs-subnav.html [hunk 1]
```diff
@@ -1,19 +1,15 @@
 <nav class="bd-subnavbar pt-2 pb-3 pb-md-2">
   <div class="container-xl d-flex align-items-md-center flex-wrap">
     <div class="d-flex align-items-center mr-sm-auto order-2 order-md-0">
-      <div class="d-none d-md-block">
+      <!-- <div class="d-none d-md-block mr-1">
```
Comment (XhmikosR) [inline site/layouts/partials/docs-subnav.html]: This whole commented out block should be removed

File: site/layouts/partials/docs-subnav.html [hunk 2]
```diff
@@ -1,19 +1,15 @@
...
         <a class="link-dark" href="/" onclick="ga('send', 'event', 'Subnav', 'click', 'Bootstrap');">Bootstrap</a>
-        {{ partial "icons/booticon-chevron-right.svg" (dict "class" "booticon d-inline-block mx-2 flex-shrink-0" "width" "12px" "height" "12px") }}
-      </div>
-
-      <a class="link-dark" href="/docs/{{ .Site.Params.docs_version }}/getting-started/introduction/" onclick="ga('send', 'event', 'Subnav', 'click', 'Documentation');">Documentation</a>
-      {{ partial "icons/booticon-chevron-right.svg" (dict "class" "booticon d-inline-block mx-2 flex-shrink-0" "width" "12px" "height" "12px") }}
```
Comment (XhmikosR) [inline site/layouts/partials/docs-subnav.html]: You should also remove the partial icon if it's not used elsewhere.

File: site/static/docs/4.3/assets/scss/_buttons.scss [hunk 3]
```diff
@@ -35,3 +35,20 @@
...
+
+.btn-bd-light {
+  color: $gray-600;
+  border-color: $gray-300;
+
+  .show > &,
+  &:hover,
+  &:active {
+    color: $bd-purple-bright;
+    background-color: #fff;
```
Comment (XhmikosR) [inline site/static/docs/4.3/assets/scss/_buttons.scss]: Please use the `$white` variable for consistency.

**Comment by XhmikosR** [comment]
I went ahead and made the changes myself. I like this patch 👍

**Review by XhmikosR — APPROVED** [review]

**Review by MartijnCuppens — APPROVED** [review]

**Comment by XhmikosR** [comment]
This has an issue on mobile, Chrome 76, Android 9.

Screenshot_20190906-091447_Chrome

**Pull request merged**